homeassistant/components/climate/tado.py

```diff
@@ -75,8 +78,11 @@ def create_climate_device(tado, hass, zone, name, zone_id):
 
     if ac_mode:
         temperatures = capabilities['HEAT']['temperatures']
-    else:
+    elif 'temperatures' in capabilities:
```

Comment:
Did the structure of the capabilities dict change? Because if not, you are now missing the `['HEAT']` part.

Reply:
No, the structure hasn't changed.  All I've changed is checking that the temperatures key exists in the provided data. Previously it was assuming it was there, and for some devices (ie, hot water control) it is not.  Obviously this then caused a KeyError as in #9530

Reply:
Oh, I misread the code. Sorry for the noise!